docs/performance-testing.md

```diff
@@ -17,7 +20,7 @@ To see a sample implementation of a test case, you can refer to `StudentProfileL
 
 ## Running Performance Tests
 
-If you want to use your own copy of [JMeter](https://jmeter.apache.org/download_jmeter.cgi), update the `test.jmeter.*` properties in `src/lnp/resources/test.properties` accordingly.  
+If you want to use your own copy of [JMeter](https://jmeter.apache.org/download_jmeter.cgi), update the `test.jmeter.*` properties in `src/lnp/resources/test.properties` accordingly.
 
 First, open the terminal and navigate to the root of project folder. Start the backend server, i.e. `./gradlew serverRun`, before running the performance tests.
 
```

Comment:
I think the link to development guide at line 27 in this file is not updated. 
Might want to update the link from `https://github.com/TEAMMATES/teammates/blob/master/docs/development.md` to `development.md`.